src/main/java/uk/gov/hmcts/reform/unspec/aspect/NoOngoingBusinessProcessAspect.java

```diff
@@ -37,7 +38,7 @@ public Object checkOngoingBusinessProcess(
         CaseEvent caseEvent = CaseEvent.valueOf(callbackParams.getRequest().getEventId());
         CaseDetails caseDetails = callbackParams.getRequest().getCaseDetails();
         CaseData caseData = caseDetailsConverter.toCaseData(caseDetails);
-        if (hasNoOngoingBusinessProcess(caseData)) {
+        if (hasNoOngoingBusinessProcess(caseData) || caseEvent.getUserType() == UserType.CAMUNDA) {
```

Comment:
could we wrap this logic in caseEvent? something like caseEvent.isCamundaEvent()

Reply:
@dharmendrak thoughts on this?

Reply:
@swalker125  done